docs/source/advanced/bgfx.rst

```diff
@@ -75,34 +75,34 @@ Configuration Settings
 |
 |	On a single window, multiple screen game, such as Darius on one physical PC monitor, specify multiple entries (one per window) like:
 |
-|		**bgfx_screen_chains hlsl,hlsl,hlsl**
+|		``bgfx_screen_chains hlsl,hlsl,hlsl``
```

Comment:
When file fragments are on their own line and indented, they don't need the `` markup to get the fixed pitch formatting and box.